packages/Carousel/src/carousel.scss

```diff
@@ -0,0 +1,130 @@
+@use '~@igloo-ui/tokens/dist/base10/variables' as tokens;
+@use '~@igloo-ui/tokens/dist/fonts';
+
+:root {
+  /* Default */
+  --ids-carousel-font-family: #{tokens.$primary-font-family};
+  --ids-carousel-font-size: #{tokens.$font-size-4};
+  --ids-carousel-nav-margin: #{tokens.$space-2} -#{tokens.$space-2} -#{tokens.$space-2};
+
+  /* Bullets */
+  --ids-carousel-bullet-color: #{tokens.$grey-300};
+  --ids-carousel-bullet-focus: #{tokens.$focus};
```

Comment:
`--ids-carousel-bullet-focus` is unused

Reply:
Thanks, I put it where it belongs on line 91